src/setupWorker/start/createRequestListener.ts

```diff
@@ -105,7 +105,7 @@ export const createRequestListener = (
 This exception has been gracefully handled as a 500 response, however, it's strongly recommended to resolve this error, as it indicates a mistake in your code. If you wish to mock an error response, please see this guide: https://mswjs.io/docs/recipes/mocking-error-responses`,
           request.method,
           request.url,
-          error,
+          error.stack ?? error,
```

Comment:
Would this not print the `error` if its stack exists?

I thought we may add a new `console.error(error)` call below this `devUtils.error()` to forward the error to the console. What do you think about this?

Reply:
This prints the error (which will just be name + message) if the stack is undefined. The stack actually also includes the name and message already. Not sure about `console.error` vs `devUtils.error`. You have to tell me what is more appropriate. For me it fits will with displaying the error directly within this message and not as a separate disconnected thing on another log.

Reply:
Oh, so the issue was that at some point there are errors that don't have a `stack`? And those were somehow ignored?

Reply:
No the issue is that `Error.toString()` only includes the `name` and `message`, never the `stack`. Per spec, `stack` on `Error` is optional, so it's best to guard against it being `undefined` (although I don't know when this could happen in practice)

Reply:
Got it. Thanks for the explanation!